final-project/styles/main.scss

```diff
@@ -1,13 +1,202 @@
 // This should be imported by the main.js file
+@import url('https://fonts.googleapis.com/css?family=Quicksand:300,400,500,700');
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700');
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600,700');
 
-// test for scss variables
-$primary-color: #2a3fbb;
+$quicksand-font: 'Quicksand', sans-serif;
+$roboto-font: 'Roboto', sans-serif;
+$open-sans-font: 'Open Sans', sans-serif;
+$titles-color: #3D4F9F;
+$titles-margin: 0px 0px 10px;
 
-* {
-  padding: 0;
-  margin: 0;
+//breakpoints
+$screen-md-min: 768px; // Tablet portrait
+$screen-lg-min: 992px; // Tablet landscape
+$screen-xl-min: 1200px; // Desktop
+
+
+// Mixins
+@mixin title-1 {
+  font-family: $quicksand-font;
+  font-size: 48px;
+  font-weight: 300;
+  color: $titles-color;
+  margin: $titles-margin; 
+  span {
+    font-weight: 500;
+  }
+}
+
+@mixin title-2 {
+  @include title-1;
+  font-size: 36px;
+}
+
+@mixin title-3 {
+  font-family: $quicksand-font;
+  font-size: 24px;
+  font-weight: 500;
+  color: $titles-color;
+  margin: $titles-margin; 
+}
+
+@mixin title-4 {
+  font-family: $roboto-font;
+  font-size: 11px;
+  font-weight: 500;
+  color: $titles-color;
+  margin: $titles-margin;
+}
+
+@mixin text {
+  font-family: $roboto-font;
+  color: #8D96BD;
+  font-size: 15px;
+  font-weight: 300;
+  text-align: center;
+  line-height: 2;
+}
+
+// Recurrent layout styles
+@mixin flex-container($width) {
+  width: $width;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+// Media queries
+@mixin md {
+  @media (min-width: #{$screen-md-min}) {
+    @content;
+  }
+}
+
+@mixin lg {
+  @media (min-width: #{$screen-lg-min}) {
+    @content;
+  }
+}
+
+@mixin xl {
+  @media (min-width: #{$screen-xl-min}) {
+    @content;
+  }
+}
+
+@mixin bg-image($url) {
+  background-image: url($url);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
 }
 
-h1 {
-  color: $primary-color;
+@mixin slider-control-dot {
+  width: 10px;
+  height: 10px;
+  display: inline-block;
+  border-radius: 50%;
+  margin: 0px 5px;
+  background-color: #D1D6E3;
+  &:hover,
+  &--selected {
+    background-color: #5283FF;
+  }
 }
+
+@mixin hover {
+  @media (hover:hover) {
+    @content;
+  }
+}
+
+// Projects Styles
+body {
+  margin:0;
+  padding:0;
+  height:100%;
+}
+
+#container {
```

Comment:
Remember, be careful when styling ids. It is commonly a bad practice because of the specificity.